src/xpk/core/pathways.py

```diff
@@ -51,7 +51,8 @@ def get_pathways_worker_args(args) -> str:
               - --xla_tpu_enable_async_collective_fusion_multiple_steps=true
               - --xla_tpu_overlap_compute_collective_tc=true
               - --xla_enable_async_all_gather=true
-              - --pathways_tmp_dir_pattern={args.pathways_gcs_location}"""
+              - --pathways_tmp_dir_pattern={args.pathways_gcs_location}
+              - --pathways_experimental_suspend_storage_mode=BACKEND_TENSORSTORE"""  # This is to enable zarr for pathways. When OCDBT is tested, remove this flag
```

Comment:
You do not need to change the PW workers here. Unless you want this flag to be consistent between the 2 jobs, you could remove this flag here.